notebooks/official/automl/sdk_automl_tabular_regression_batch_bq.ipynb

```diff
@@ -74,6 +74,14 @@
         "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.                                        pyarrow \
</pre><p>Is this needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Updated pip installs. removed pyarrow. Added bqstorage to 'google-cloud-bigquery[bqstorage,pandas]'</p>